### PR TITLE
Add sizes option or responsive images as of issue #159

### PR DESCRIPTION
**Pull request by Sommerregen**
This is the pull request of issue #159, adding "sizes" option to media actions and bugfixing broken responsive image paths for Windows systems.

### CODE REVIEW

**Comment by rhukster** [comment]
Thanks for the PR. I have chatted with Gert and we are going to manually merge this into his media enhancement branch next week. Thanks

**Comment by Sommerregen** [comment]
Ok, thanks. If I can do something, then let me know.

File: system/defines.php [hunk 1]
```diff
@@ -7,7 +7,8 @@
 
 // Directories and Paths
 if (!defined('GRAV_ROOT')) {
-    define('GRAV_ROOT', getcwd());
+		// Normalize root path (i.e. on Windows convert backslashes to slashes)
+    define('GRAV_ROOT', str_replace(DIRECTORY_SEPARATOR, DS, getcwd()));
```
Comment (Gertt) [inline system/defines.php]: @rhukster I am not gonna touch this for media branch. Should I create a separate issue for it?

**Comment by Sommerregen** [comment]
I see, my code is now in core Grav +0.9.22.

**Pull request closed without merging**